WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -72,6 +72,7 @@
 
 int ddLogLevel                                                  = LOG_LEVEL_INFO;
 static NSString * const kUsageTrackingDefaultsKey               = @"usage_tracking_enabled";
+static NSString * const MustShowWhatsNewPopup = @"MustShowWhatsNewPopup";
```

Comment:
Fair enough, does look a bit off!

Changed it "Pfhor Your Eyes Only".  (If you happen to know that reference you have my respect O.O)